code/_onclick/hud/alert.dm

```diff
@@ -177,6 +177,41 @@
 	desc = "You're severely malnourished. The hunger pains make moving around a chore."
 	icon_state = "starving"
 
+///Vampire "hunger"
+
+/obj/screen/alert/fat/vampire
+	name = "Fat"
+	desc = "You drank too much blood, somehow, lardass. Run around the station and lose some weight."
```

Comment:
Extraneous comma - should be "You drank too much blood somehow, lardass." or "You somehow drank too much blood, lardass."

Reply:
Updated to reflect suggestion.